apps/web/components/Charts/v2/renderers/ChartRenderer.tsx

```diff
@@ -39,6 +39,64 @@ type ChartProps = {
   slice: IChart
 }
 
+const transformData = (items: any) => {
+  return {
+    statisticsForHeader: items
+      .flatMap((item: { categoryItems: any }) => item.categoryItems)
+      .map((item: { id: string; value: string }) => ({
+        key: item.id,
+        value: parseInt(item.value),
+      })),
+  }
+}
+
+const getData = (slice: IChart, queryResult: DataItem[]): any => {
+  const base = slice.sourceData ? JSON.parse(slice.sourceData) : queryResult
+
+  const values = slice.components.map((component) =>
+    JSON.parse(component.values ?? '{}'),
+  )
+
+  if (values[0]?.typeOfSource === 'manual') {
+    const sourceDataType = values[0]?.typeOfManualDataKey
+    const componentType = slice.components[0]?.type
+
+    let transformedData
```

Comment:
Can we skip defining an empty variable since we can just instead return directly in both cases